backend/src/schema/resolvers/reports.spec.js

```diff
@@ -1,37 +1,55 @@
 import { GraphQLClient } from 'graphql-request'
```

Comment:
would love to refactor this test to remove the use of `graphql-request`, but maybe that's not your priority at the moment... we could help with that, @aonomike is getting awesome at this... maybe in a separate `PR` though?

Reply:
You are right @mattwr18 . I set the priority to get this PR done. Would love if @aonomike jumps into that … 😍